contents/blog/turning-engineers-into-product-people.md

```diff
@@ -0,0 +1,40 @@
+---
+date: 2021-05-28
+title: How to turn your engineers into product people
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+By: [Mohammed Shehu](https://twitter.com/shehuphd), Content Lead @ PostHog
+
+The worst thing you can do to an engineer is micromanage their every move. Engineers are inherent problem solvers who have as much interest in fixing a failing product (or improving a good one) as any product manager is. Not giving an engineer the space to contribute ideas and execute solutions spells death for any startup with grand ambitions because the best engineers can simply leave and join a company that’ll give them the autonomy they seek.
+
+At PostHog, one of our core values is autonomy. New devs and non-techies who start at PostHog are usually surprised by the lack of micromanagement, onerous KPIs, and death by meetings. This is deliberate, as people can only do their best work when given the freedom to do so. We run two-week sprints where everyone decides what they’re going to work on for the next 14 days and run a retrospective session at the end of each sprint. Not only is more impactful work done this way, but new problems are also discovered and solved during each sprint that weren’t originally planned for. For example, two of our main features - [session recording](https://posthog.com/product-features/session-recording) and [feature flags](https://posthog.com/product-features/feature-flags) - were ideated during one of our hackathons at a company offsite. The whole concept of our hackathons revolves around experimentation, and when you give engineers the freedom to play around with new ideas, magical things happen.
```

Comment:
micromanagement -> I'd link to https://posthog.com/handbook/company/management
death by meetings -> I'd link to https://posthog.com/blog/meetings